extension/opampextension/README.md

```diff
@@ -0,0 +1,46 @@
+# OpAMP Agent Extension
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]  |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aextension%2Fopamp%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aextension%2Fopamp) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aextension%2Fopamp%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aextension%2Fopamp) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@portertech](https://www.github.com/portertech), [@evan-bradley](https://www.github.com/evan-bradley), [@tigrannajaryan](https://www.github.com/tigrannajaryan) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+## Configuration
+
+The following settings are required:
+
+- `server`: The OpAMP server connection settings.
+  - `ws`: The OpAMP websocket transport settings.
+    - `endpoint` (no default): The OpAMP server websocket endpoint (URL).
+
+The following settings are optional:
+
+- `server`: The OpAMP server connection settings.
+  - `ws`: The OpAMP websocket transport settings.
+    - `tls`: TLS settings.
+    - `headers`: HTTP headers to set.
+- `instance_uid`: A ULID formatted as a 26 character string in canonical
```

Comment:
ULID?

Reply:
Maybe link to https://github.com/ulid/spec to disambiguate?